code-challenges.js

```diff
@@ -31,18 +68,34 @@ var myNumbers1 = [3, 7, 0, 6, -9]
 
 var myNumbers2 = [8, -7, 0, 6, 2, 9, 13]
 
-
-
-
+//input numbers 
+//output numbers 
+// this is where i make my constant declaration 
+const oddNumbers = (numbers) => {
+    let num =[]
+     //this is where my loop will be to make the numbers odd using module to get the odds numbers
+    for(let i=0; i<numbers.lenght; i++){
+        if(number[i] % 2 ! =0){
+            num.push(array[i])
+        }
+    }
+    return num
+}
+console.log(oddNum(myNumbers2))
+
+//kept getting syntax error for the bang character couldn't figure out how to make it work
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
+const noVowels =(vowel) = >{
+if(1=e)
+else(e=0)
+}
 
-
-
+// Tried to work on it with Diego we looked through stack overflow, and  compared our notes we couldnt figure out how to make the function work. I believe that we have to creat a loop with conditionals and and the shift funtion but i'm not sure.
```

Comment:
Did you have a chance to work through this problem during our assessment review?